### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by salmanthecoder**

### CODE REVIEW

File: src/lm-profile-status.html [hunk 1]
```diff
@@ -1,48 +0,0 @@
...
-               >
-    </iron-ajax>
-    <span>{{ajaxResponse.responseMessage}}</span>
-  </template>
-  <script>
-    class ProfileStatus extends Polymer.Element {
-      static get is() { return 'lm-profile-status'; }
-      static get properties() {
-        return {
-          prop1: {
```
Comment (manhar-developer) [inline src/lm-profile-status.html:21]: Remove prop1 and provide meaningful names to the variable

File: src/my-app.html [hunk 2]
```diff
@@ -90,28 +96,28 @@
 
     <app-drawer-layout fullbleed narrow="{{narrow}}">
       <!-- Drawer content -->
-      <app-drawer id="drawer" slot="drawer" swipe-open="[[narrow]]">
-        <app-toolbar>Menu</app-toolbar>
+      <!-- <app-drawer id="drawer" slot="drawer" swipe-open="[[narrow]]">
```
Comment (manhar-developer) [inline src/my-app.html:99]: Remove commented code

File: src/lm-profile-status.html (hunk 1 above)
Comment (salmanthecoder) [reply src/lm-profile-status.html:21]: Ok I ll take care

File: src/my-app.html (hunk 2 above)
Comment (salmanthecoder) [reply src/my-app.html:99]: Ok I ll remove

**Pull request merged**